### PR TITLE
configure.ac: fix header checks for macOS

### PR DESCRIPTION
**Pull request by barracuda156**
These headers are available, but they won’t be found on earlier macOS versions due to requirement for inclusion order.

### CODE REVIEW

**Comment by DimitriPapadopoulos** [comment]
Out of curiosity, which macOS versions fail?

**Comment by barracuda156** [comment]
> Out of curiosity, which macOS versions fail?

AFAIK, < 10.9

**Comment by DimitriPapadopoulos** [comment]
Not exactly recent macOS versions, they have reached EOL in 2015. But then why not support them if minimal changes are required.

File: configure.ac [hunk 1]
```diff
@@ -86,7 +86,7 @@ util.h \
 ])
 
 # Checks for optional header files with prerequisites of other headers.
-AC_CHECK_HEADERS([net/if_var.h],[],[],[#include <net/if.h>])
```
Comment (DimitriPapadopoulos) [inline configure.ac:89]: After #701, `net/if_var.h` is not needed any more . You may remove this paragraph altogether.

Comment (barracuda156) [reply configure.ac:89]: Ok, I will rebase today, thanks.

Comment (DimitriPapadopoulos) [reply configure.ac]: I solved the conflict and merged from the web interface. Thank you very much for this improvement of portability @barracuda156.

Comment (barracuda156) [reply configure.ac]: Thank you! (Sorry, I got distracted by other stuff.)

**Pull request merged**